src/pushsource/_impl/backend/staged/staged_cloud.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import os
+import yaml
+
+from .staged_base import StagedBaseMixin, handles_type
+from ...model import VHDPushItem, AmiPushItem, AmiRelease, BootMode, KojiBuildInfo
+
+LOG = logging.getLogger("pushsource")
+
+
+class StagedCloudMixin(StagedBaseMixin):
+    def __build_ami_push_item(self, resources, src, origin, name):
+        build_resources = resources.get("build")
+        release_resources = resources.get("release", {})
```

Comment:
As this will crash later if `release` is present but null, I suggest this more robust method of setting the default:

```suggestion
        release_resources = resources.get("release") or {}
```